lib/Timber.php

```diff
@@ -157,6 +157,75 @@ public static function get_posts( $query = false, $PostClass = 'Timber\Post', $r
 		return PostGetter::get_posts($query, $PostClass, $return_collection);
 	}
 
+	/**
+	 * Gets a post by post slug.
+	 *
+	 * @api
+	 * @since 2.0.0
+	 * @example
+	 * ```
+	 * $post = Timber::get_post_by_slug( 'about-us' );
+	 * ```
+	 *
+	 * @param string       $slug      The post slug to search for.
+	 * @param string|array $post_type Optional. The post type or an array of post types to look
+	 *                                for. Default `any`, which will look for posts in any post
+	 *                                type.
+	 *
+	 * @return \Timber\Post|null A Timber post or null if no post could be found.
```

Comment:
`null` ➡️ `false`

Reply:
Resolved in https://github.com/timber/timber/pull/2169/commits/5ae15e53ce9d194199a63f2d328a41f37fd5b6a4.